### DISCUSSION TITLE
which models are using feat_dynamic_features

### DISCUSSION THREAD

**Discussion by SaeidShamsaliei** [awslabs/gluonts#925] [Q&A]
Hi,

I thought only DeepAR and DeepState are supporting the feat_dynamic_features since they have ‘use_feat_dynamic_real’ and others just ignore it. How ever, it was mentioned in #693 that many models support that. Which models are currently using feat_dynamic_features ?

Thank you so much in advance.

**Comment by Roxyi** [comment]
Same question. Many models don't have the parameter `use_feat_dynamic_real`. Does it mean that they use `feat_dynamic_real` by default?
